MSBuild.sln

```diff
@@ -1,8 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30413.136
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
+MinimumVisualStudioVersion = 17.0.31903.59
```

Comment:
Using `save as…` within Visual Studio IDE causes it to revert to older version. May be VS bug? 🧐

Reply:
> save as…

When would you Save As on a solution file?